shared/src/main/scala/com/cibo/evilplot/plot/BoxPlot.scala

```diff
@@ -113,28 +201,9 @@ object BoxPlot {
         summaries,
         boxRenderer.getOrElse(BoxRenderer.default()),
         pointRenderer.getOrElse(PointRenderer.default()),
-        spacing.getOrElse(theme.elements.boxSpacing)
+        spacing.getOrElse(theme.elements.boxSpacing),
+        clusterSpacing
       )
     )
   }
-
-  /** Create a box plots for a sequence of distributions.
-    * @param data the distributions to plot
-    * @param boxRenderer the `BoxRenderer` to use to display each distribution
-    * @param pointRenderer the `PointRenderer` used to display outliers
-    * @param quantiles quantiles to use for summary statistics.
-    *                  defaults to 1st, 2nd, 3rd quartiles.
-    * @param spacing spacing how much spacing to put between boxes
-    * @param boundBuffer expand bounds by this factor
-    */
-  @deprecated("use apply", "2018-03-28")
-  def custom(
-    data: Seq[Seq[Double]],
```

Comment:
depending on whether we release this as part of 0.5.0 or 0.4.2 we may want/need to keep this for now.